src/components/NewLocationPage/Shared/SectionContainer.tsx

```diff
@@ -0,0 +1,12 @@
+/**
+ * Parent div for each module on the new location page
+ */
+
+import React from 'react';
+import { Container } from './Shared.style';
+
+const SectionContainer: React.FC = ({ children }) => {
+  return <Container>{children}</Container>;
+};
+
+export default SectionContainer;
```

Comment:
This component is not needed, the Container already received `children` + all the other props. Consider just export it with a name from `NewLocationPage/Shared/index.ts`


```tsx
export { Container as SectionContainer } from './Shared.style';
```

Reply:
oooo yes this is very true.